pkg/authorization/bootstrap/policy.go

```diff
@@ -36,6 +40,19 @@ const (
 func clusterRoleBindings() []rbacv1.ClusterRoleBinding {
 	return []rbacv1.ClusterRoleBinding{
 		clusterRoleBindingCustomName(rbacv1helpers.NewClusterBinding("cluster-admin").Groups(SystemKcpClusterWorkspaceAdminGroup, SystemKcpAdminGroup).BindingOrDie(), "system:kcp:clusterworkspace:admin"),
+		clusterRoleBindingCustomName(rbacv1helpers.NewClusterBinding("system:kcp:tenancy:reader").Groups(SystemKcpClusterWorkspaceAccessGroup).BindingOrDie(), "system:kcp:clusterworkspace:access"),
```

Comment:
so this is saying, `system:kcp:clusterworkspace:access` group will be assigned to `system:kcp:tenancy:reader` role, which allows `list` on `clusterworkspaces/workspace`

Reply:
which users are assigned to `system:kcp:clusterworkspace:access` group?

Reply:
system:kcp:clusterworkspace:access